TINY-6363: Fixed the root document able to be scrolled when a dialog is open and rendered inside a shadow root (#6123)
tinymce/tinymce: modules/tinymce/changelog.txt

Version 5.6.0 (TBD)

>     Fixed the root document able to be scrolled when a dialog is open and rendered inside a shadow root #TINY-6363

Reviewer: This reads a little odd.

Maybe

```suggestion
    Fixed the root document being able to scroll when a dialog is open and rendered inside a shadow root #TINY-6363
```

Author: `being` is in our list of words we're not allowed to use in changelogs/docs, so can't do that sorry.

Reviewer 2: This might not be any better, but would `Fixed the root document incorrectly scrolling when a dialog...` work?

Author: That's nicer to read, but it has a different meaning, so I can't change to that sorry. That implies it scrolls without user interaction.

Reviewer 3: Doesn't have the word "Fixed" but what about "Prevented scrolling in the root document when a dialog is open and rendered inside a shadow root"

Author: Needs to start with `Fixed` as you already know :stuck_out_tongue: and between that requirement and our banned words list, this is just really hard to come up with something nice.

Author: ```suggestion
    Fixed an issue where the root document could be scrolled while an editor dialog was open inside a shadow root #TINY-6363
```
